src/main/cheat_code.cpp

```diff
@@ -1775,7 +1775,7 @@ static void holyTrinity()
         NewEffect(10, tempLocation);
     }
     
-    if HolyTrinity
+    if (HolyTrinity)
```

Comment:
Little note: in the project's code style we don't have the space between if and paren. So, you should write:
```cpp
if(HolyTrinity)
```
Please follow the code style of projects that you contribute.